src/sentry/utils/committers.py

```diff
@@ -198,6 +198,8 @@ def get_event_file_committers(project, event, frame_limit=25):
     # file paths.
     if event.platform == 'java':
         for frame in frames:
+            if frame.get('filename') is None:
+                continue
             if '/' not in frame.get('filename') and frame.get('module'):
```

Comment:
Can we just use `frame.get('filename', '')` instead?

Reply:
Nm, that won't work

Reply:
Yap, be careful there will be explicit nulls in there after normalization.